src/yardstick/cli/config.py

```diff
@@ -19,7 +19,7 @@
 
 @dataclass()
 class Profiles:
-    data: Dict[str, Dict[str, str]] = field(default_factory=dict)
+    data: dict[str, dict[str, str]] = field(default_factory=dict)
```

Comment:
I'm sure the answer is "because python," but why did this change case? Is this a built in type that doesn't need to be imported now?

Reply:
Correct. Last I used type hints in python heavily (~3.7) I needed to use the collections provided in the `typing` module (e.g. `from typing import Dict`  then use `Dict`). Since then type hinting doesn't require these collections and instead the standard collections in python can be used in their place (starting with https://peps.python.org/pep-0585/ ), which is preferred. 

I want to say this was a linter-induced change, but can't remember the specifics here.